pkg/get/tools.go

```diff
@@ -2545,5 +2545,29 @@ https://github.com/{{.Owner}}/{{.Repo}}/releases/download/{{.Version}}/{{.Name}}
 						{{.Name}}_{{slice .Version 1}}_{{$osStr}}_{{$archStr}}.tar.gz`,
 		})
 
+	tools = append(tools,
+		Tool{
+			Owner:       "golangci",
+			Repo:        "golangci-lint",
+			Name:        "golangci-lint",
+			Description: "Go linters aggregator.",
+			BinaryTemplate: `
+							{{$osStr := ""}}
+							{{ if HasPrefix .OS "ming" -}}
+							{{$osStr = "windows"}}
+							{{- else if eq .OS "linux" -}}
+							{{$osStr = "linux"}}
+							{{- else if eq .OS "darwin" -}}
+							{{$osStr = "darwin"}}
+							{{- end -}}
+							{{$archStr := .Arch}}
+							{{- if eq .Arch "aarch64" -}}
```

Comment:
In case of a `darwin`, this isn't `aarch64` but `arm64` can you add that as an `or` to to your condition please?

Reply:
Added and clean up naming a bit.